src/main/java/teammates/common/util/Const.java

```diff
@@ -817,6 +817,8 @@ private Const() {
         public static final String RESPONDENT_IS_INSTRUCTOR = "respondentisinstructor";
         public static final String RESPONDENT_IS_TO_BE_REMOVED = "respondentistoberemoved";
 
+        public static final String RECOVERY_EMAIL = "recoveryemail";
```

Comment:
Why we have to change everything in this late?.... I thought backend development start first. 
Any rationale behind the change? Otherwise I suggest stick with current value as it is also the one suggested in the specification.

Reply:
Actually, thinking about it again, I feel `recover-session-links` is the most appropriate (as I stated a while earlier, actually). I don't mind refactoring everything if we get this PR merged first. Otherwise, I can revert to using `LinkRecovery` again in the front end.

I guess the main reason for including `session` would be that it's more specific. Links can potentially refer to anything (like join URL, for example).
And `link**s**` because it's not a "recover password" thing where you get 1 URL. The email consists of multiple links.

Reply:
Yes, I agree adding `session` make things clear. Shall we agree on `recover-session-links`?

Reply:
I think SessionLink looks okay to me, I will update shortly.